pages/team.vue

```diff
@@ -71,13 +153,33 @@ import ResizeImg from '@/components/widgets/ResizeImg.vue'
 
 export default {
   components: { PageLoader, ResizeImg },
+  data() {
+    return {
+      dialog: false,
+      selectedMember: {
+        name: '',
+        profile_info: {
+          about: '',
+          batch: '',
+          cv: '',
+          interests: []
+        },
+        thumbnail: '',
+        uid: ''
+      }
```

Comment:
Why these? Why not null?

Reply:
Was using null initially but was told to review the code from the projects section.

Reply:
Should I change it to null?

Reply:
Yes